include/bluetooth/mcc.h

```diff
@@ -26,259 +34,321 @@ extern "C" {
 
 /**** Callback functions ******************************************************/
 
-/** @brief Callback function for bt_mcc_discover_mcs()
+/**
+ * @brief Callback function for bt_mcc_discover_mcs()
+ *
+ * Called when a media control server is discovered
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  */
 typedef void (*bt_mcc_discover_mcs_cb)(struct bt_conn *conn, int err);
 
-/** @brief Callback function for bt_mcc_read_player_name()
+/**
+ * brief Callback function for bt_mcc_read_player_name()
```

Comment:
A lot of these lines are missing a @ in front of the `brief`:
```suggestion
 * @brief Callback function for bt_mcc_read_player_name()
```

Reply:
All of them, I think - the result of a typo in a search and replace (search and destroy ...)

Fixed